app/src/main/java/de/westnordost/streetcomplete/quests/way_lit/WayLit.kt

```diff
@@ -5,4 +5,5 @@ enum class WayLit(val osmValue: String) {
     AUTOMATIC("automatic"),
     YES("yes"),
     NO("no"),
+    IS_ACTUALLY_STEPS(""),
```

Comment:
This looks like a hack. Better use sealed classes?

Reply:
Done in [`65b2244` (#3334)](https://github.com/streetcomplete/StreetComplete/pull/3334/commits/65b2244c4efb11484c8f69de2adb76f903467d66)